admin/configuration/default-interface.md

```diff
@@ -0,0 +1,27 @@
+# Default user interface
+
+The 2i2c hubs offer the following user interfaces by default:
+
+## Jupyter Notebook (Classic)
+
+What many people mean when they say 'Jupyter', this familiar interface is used
```

Comment:
I think we should avoid saying things like "what most people think of" - we don't know what kind of mental model users are gonna bring with them. I'd suggest we just keep things informative about the interface itself. E.g. "The original single-document interface for creating Jupyter Notebooks."